### ISSUE TITLE
Underline markdown change

### ISSUE THREAD

**Issue by StrangerGithuber** [erusev/parsedown#749]
Hello
How can I make that the `__Something__` will not be bold text but underlined text?
Because when I am using the script then it shows as bold text. 
I created a Discord chat bot and I am trying to create a dashboard for that. I am using turndown script to convert and save in the database the markdown text and as editor I am using the tinymce.
The discord app recognize the `__Something__` markdown text as an underlined text but this parsedown recognize as a bold text.
How can I can change?

**Comment by MerryPanda** [comment]
I agree, it is more logical to use __word__ for underlines. No need for multiple tags with the same effect

:D

**Comment by JohnKlein985** [comment]
One year later and underline is still not possible...?
